bgpd/bgpd.h

```diff
@@ -746,6 +746,8 @@ struct bgp {
 	/* Process Queue for handling routes */
 	struct work_queue *process_queue;
 
+	bool fast_convergence;
```

Comment:
I don't remember what was discussed, but can't this be enabled by default?

Reply:
Original discussion is part of #8251 . This behaviour change might surprise the operators, so It was decided to have a knob to enable it.